src/main/java/io/eroshenkoam/xcresults/export/ExportCommand.java

```diff
@@ -190,10 +206,12 @@ private void exportTestSummary(final ExportMeta meta, final JsonNode testSummary
         if (test.has(SUMMARY_REF)) {
             final String ref = test.get(SUMMARY_REF).get(ID).get(VALUE).asText();
             summaries.add(getReference(ref));
+        } else {
+            if (test.has(TYPE) && test.get(TYPE).get(NAME).textValue().equals("ActionTestMetadata")) {
+                summaries.add(test);
+            }
         }
-        if (test.has(TYPE) && test.get(TYPE).get(NAME).textValue().equals("ActionTestMetadata")) {
```

Comment:
I believe the introduction of unit test support has resulted in duplicate tests being exported in JSON. When allure report is generated, the number of tests shown are double.